data/data/ovirt/variables-ovirt.tf

```diff
@@ -19,6 +19,21 @@ variable "ovirt_password" {
   description = "The plain password of user to access Engine API"
 }
 
+variable "ovirt_cafile" {
```

Comment:
I'm pushing to removing this variable and have it in the bundle so I think you can remove it